src/shader_recompiler/ir/opcodes.h

```diff
@@ -21,7 +21,7 @@ namespace Detail {
 struct OpcodeMeta {
     std::string_view name;
     Type type;
-    std::array<Type, 5> arg_types;
+    std::array<Type, 6> arg_types;
```

Comment:
You also need to update args array in Inst class, see value.h

Reply:
Done